wrapping/python/openmeeg/tests/test_python.py

```diff
@@ -53,17 +53,18 @@ def test_python(data_path, tmp_path):
     n_eeg_sensors = patches.getNumberOfSensors()
 
     # Compute forward problem (Build Gain Matrices)
-    # gauss_order = 3  # XXX cannot get Integrator exposed
+    # gauss_order = 3  # XXX Integrator is now exposed, just need to use it...
```

Comment:
@agramfort, in case you want to take a stab at this, it should be available after this PR is in

Reply:
I like the idea of adding a control over verbosity. I like less the need to pass it as a constructor parameter.
Could we instead add a set_verbose method that allows to change the verbosity at will.
We could even try to make different default for different modes (off for python and on for C++ apps to keep backward compatibility)....

Any opinion ?

Reply:
In part the motivation behind having it in the constructor was because it was already this way elsewhere in the codebase

At least for HeadMat, it has to be in the constructor because all the computations are done immediately when you call the constructor